.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,10 @@
+name: Build docker image
+
+on: pull_request
+
+jobs:
```

Comment:
You'll need to add a job to parse the version from the `Dockerfile` to be able to have the version so you can push it above.